bouncer/src/repo/core/handler/fileservice/repo_file_handler_fs.cpp

```diff
@@ -112,17 +112,17 @@ std::vector<std::string> FSFileHandler::determineHierachy(
 }
 
 std::string FSFileHandler::uploadFile(
-	const std::string          &database,
-	const std::string          &collection,
-	const std::string          &keyName,
-	const std::vector<uint8_t> &bin
+	const std::string& database,
```

Comment:
Are you using a code cleaner that is doing this?

Can we revert? Imo & should be closer to the variable (also you're making it inconsistent with the rest of the code base)

This applies to the other PR as well.

Reply:
As per the suggestion I have turned off auto format in my visual studio due to which extra space was added in the code.